electrum builds fails while producing gitian binaries

```
2019-06-01 13:40:54,718.INFO:    Compiling librocksdb-sys v5.17.2
2019-06-01 13:41:52,723.INFO: warning: redundant linker flag specified for library `stdc++`
2019-06-01 13:41:52,723.INFO: 
2019-06-01 13:41:52,864.INFO:    Compiling rocksdb v0.12.1
2019-06-01 13:41:53,833.INFO:    Compiling electrs v0.6.1 (/home/ubuntu/build/bitcoin/distsrc-x86_64-linux-gnu/electrs)
2019-06-01 13:41:54,141.INFO: error[E0658]: use of unstable library feature 'try_from' (see issue #33417)
2019-06-01 13:41:54,141.INFO:  --> src/util.rs:5:5
2019-06-01 13:41:54,141.INFO:   |
2019-06-01 13:41:54,141.INFO: 5 | use std::convert::TryInto;
2019-06-01 13:41:54,141.INFO:   |     ^^^^^^^^^^^^^^^^^^^^^
2019-06-01 13:41:54,142.INFO: 
2019-06-01 13:41:54,810.INFO: error[E0658]: use of unstable library feature 'try_from' (see issue #33417)
2019-06-01 13:41:54,811.INFO:   --> src/util.rs:25:10
```

--- a/contrib/electrs/build_electrs.py
+++ b/contrib/electrs/build_electrs.py
@@ -5,8 +5,8 @@ import os
 import sys
 import shutil
 GIT_REPO = "https://github.com/dagurval/electrs.git"
-GIT_BRANCH = "v0.6.1bu"
-EXPECT_HEAD = "175c1d161b5d30c60a20b5b9922e65fd05e6b59c"
+GIT_BRANCH = "v0.7.0bu"
+EXPECT_HEAD = "5d37e7fe52e5cd81c00f269ad09dd32a835ff808"
 
 ROOT_DIR = os.path.realpath(
         os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))

